proto/cacheclient.proto

```diff
@@ -370,6 +389,23 @@ message _SetContainsResponse {
     _Missing missing = 2;
   }
 }
+message _SetLengthRequest {
```

Comment:
You have an extra line on the mr2rs:

![image](https://github.com/momentohq/client-protos/assets/127151605/aa324cd4-5059-4925-83c3-a434d9ed46e4)

can you add it here too?